lib/v4/migration-helpers/update-folder-structure.js

```diff
@@ -0,0 +1,25 @@
+const { resolve, join, basename } = require('path');
+const { logger } = require('../../global/utils');
+
+const moveDir = (appPath, folder, srcDir) => {
+	try {
+    await fs.copy(join(appPath, folder), srcDir);
+  } catch (error) {
+    logger.info(`${basename(appPath)}/${folder} not found, skipping...`);
+  }
+}
+
+module.exports = async (appPath) => {
+  const strapiAppPath = resolve(appPath);
+	const srcDir = join(strapiAppPath, 'src')
+
+	moveDir(strapiAppPath, 'admin', srcDir)
+	moveDir(strapiAppPath, 'components', srcDir)
+	moveDir(strapiAppPath, 'extensions', srcDir)
+	moveDir(strapiAppPath, 'middlewares', srcDir)
+	moveDir(strapiAppPath, 'plugins', srcDir)
+	moveDir(strapiAppPath, 'policies', srcDir)
+  
+  logger.info(`migrated remaining folder structure to Strapi v4 🚀`);
```

Comment:
similar comment, it would be better if this logs before all the `git` logs in `lib/v4/migration-helpers/update-api-folder-structure.js`, maybe we could just replace: 

```
logger.info(`migrated ${chalk.yellow(basename(strapiAppPath) + '/api')} to Strapi v4 🚀`);
```
with something like:

```
logger.info(`migrated ${chalk.yellow(basename(strapiAppPath))} to Strapi v4 🚀`);
```